src/importlinter/application/use_cases.py

```diff
@@ -298,6 +302,20 @@ def _get_include_external_packages(user_options: UserOptions) -> Optional[bool]:
     return include_external_packages_str in ("True", "true")
 
 
+def _get_exclude_type_checking_imports(user_options: UserOptions) -> bool:
+    """
+    Get a boolean (or None) for the include_external_packages option in user_options.
```

Comment:
This docstring needs correcting, it's been copied over from the other function.

Reply:
shame on me 😅